model_analyzer/config/generate/search_parameters.py

```diff
@@ -73,6 +75,9 @@ def get_list(self, name: str) -> Optional[List[Any]]:
     def _populate_search_parameters(self) -> None:
         if self._parameters:
             self._populate_parameters()
+        else:
+            # Always populate batch size if nothing is specified
```

Comment:
What is the background for this? Is it just batch size that needs to be specified? Should this function name be more generic?

Reply:
If the user doesn't specify any parameters then we just need to populate batch sizes (for now). This will change when we add the option for the user to have optuna search concurrency as well.

Reply:
Can you explain how it will change?
My main point is that it seems like the function should be named something more generic (and for now it only adds batch_size, but in the future it may do more or less or something else)

Reply:
Updated to make this a call to a more generic method